src/components/Navbar.js

```diff
@@ -83,9 +34,10 @@ const Navbar = () => {
             </span>
           </button>
         )}
-        <p className="text-xl">{setTitle()}</p>
+        <span className="text-xl">Country Pulse</span>
       </nav>
     </>
   );
 };
+
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍